cmd/romie.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
```

Comment:
If we agree on separating named imports from non-named, let's rearrange them as:

```suggestion
	"github.com/sirupsen/logrus"
	"github.com/spf13/cobra"
	"github.com/spf13/viper"

	homedir "github.com/mitchellh/go-homedir"
```

The order I usually prefer to use is, alphabetically sorted inside blocks and blocks order:
1. builtin
2. named builtin
3. external
4. named external
5. local
6. named local

```go
import (
  "fmt"
  "strings"
  "strconv"

  fp "path/filepath"

  "github.com/spf13/cobra"
  "github.com/spf13/viper"

  homedir "github.com/mitchellh/go-homedir"
  log "github.com/sirupsen/logrus"

  "github.com/romie-gr/romie/cmd"

  utilities "github.com/romie-gr/romie/utils"
)

Let's discuss more if you want.